src/structures/Bot.js

```diff
@@ -47,7 +47,7 @@ module.exports = class Bot extends Client {
          *
          * @type {Database}
          */
-    if (this.database !== undefined) {
```

Comment:
Also uhh? Won't this always be undefined that way?

Reply:
I see this... I will remove it in the next commit.